packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -169,14 +169,12 @@ const start = async (zcf, privateArgs) => {
     { addLiquidityToAmmPool },
   );
 
-  /** @typedef {typeof creatorFacet} ReserveCreatorFacet */
```

Comment:
Did this not work?

Reply:
No, it didn't.  The typedefs were scoped to the function they appeared in, so they weren't reachable from other modules.